CONTRIBUTING.md

```diff
@@ -16,4 +16,4 @@ on the projects add to it. For your project to be added, it is important that:
 Please note that we are going to consider removing a non-working snap from the
 repo if we can't get it to work after a month without updates.
```

Comment:
This is in the contextual diff, I'll do a direct typo fix following this PR, nice addition! :)